<?xml version="1.0"?>
<launch>
    <node pkg="group1_roslab" name="lidarScanner" type="lidarScanner.py" output="screen"/>
    <node pkg="group1_roslab" name="controller" type="controller.py" output="screen"/>
    <node pkg="group1_roslab" name="keyboardTalker" type="keyboardTalker.py" output="screen"/>
    <include file="$(find racecar)/launch/teleop.launch" />

    <!-- only for Simulation -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find group1_roslab)/cfg/config_file.rviz"/>
</launch>

